log_miner.py

```diff
@@ -36,8 +37,15 @@ def __init__ (self,threadNum):
     self.t = threadNum
  
   def run(self):
-    #dsn = cx_Oracle.makedsn('127.0.0.1', 1521, 'ORCL') 
-    conn = cx_Oracle.connect('root', 'BiouTaSeCtOmPavA', '(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=127.0.0.1)(PORT=1521))(CONNECT_DATA=(SID=ORCL)))')
+    #dsn = oracledb.makedsn('127.0.0.1', 1521, 'ORCL')
+    conn_config = {
+        'user': 'root',
+        'password': 'BiouTaSeCtOmPavA',
+        'dsn': '(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=127.0.0.1)(PORT=1521))(CONNECT_DATA=(SID=ORCL)))'
+    }
```

Comment:
What's this? Just wondering where these credentials came from and whether they're lying around from testing?

Reply:
Similar to above, this is legacy code which has come through from original tap code. I believe someone had a test database for the tap tests (could be wrong) for log_miner.

Here is the original tap code - https://github.com/singer-io/tap-oracle/blob/master/log_miner.py

Reply:
I will leave this legacy code for now, but worth looking at long-term for a refactor.